src/buffer/out/AttrRow.cpp

```diff
@@ -11,8 +11,8 @@
 // - attr - the default text attribute
 // Return Value:
 // - constructed object
-ATTR_ROW::ATTR_ROW(const uint16_t width, const TextAttribute attr) :
-    _data(width, attr) {}
+ATTR_ROW::ATTR_ROW(const til::CoordType width, const TextAttribute attr) :
+    _data(gsl::narrow<uint16_t>(width), attr) {}

```

Comment:
NTS: Leonard tells me that the additional branches for narrowing checks here don't harm performance